addons/humanizer/scripts/services/equipment.gd

```diff
@@ -14,11 +14,23 @@ static func load_mesh_arrays(equip:HumanizerEquipmentType):
 	sf_arrays[Mesh.ARRAY_VERTEX].fill(Vector3.ZERO)
 	return sf_arrays
 
-static func fit_mesh_arrays(mesh_arrays:Array, helper_vertex_array: PackedVector3Array, mhclo: MHCLO) -> Array: 
+static func fit_mesh_arrays(mesh_arrays:Array, helper_vertex_array: PackedVector3Array, mhclo: MHCLO) -> Array:
 	var clothes_scale =	mhclo.calculate_mhclo_scale(helper_vertex_array)
+	#if mhclo.mh2gd_index.size() == 0:
+		#assert(mhclo.vertex_data.size() % 3 == 0)
 	for mh_id in mhclo.vertex_data.size():
 		var vertex_line = mhclo.vertex_data[mh_id]
-		var new_coords = mhclo.get_mhclo_vertex_position(helper_vertex_array,vertex_line,clothes_scale)
+		var new_coords = mhclo.get_mhclo_vertex_position(helper_vertex_array, vertex_line, clothes_scale)
+		#if mh_id >= mhclo.mh2gd_index.size():
+			#continue
+		#if is_nan(new_coords.x) or is_nan(new_coords.y) or is_nan(new_coords.z):
+			#new_coords = Vector3()
+			#continue
+		assert(not is_nan(new_coords.x))
+		assert(not is_nan(new_coords.y))
+		assert(not is_nan(new_coords.z))
+		if mh_id >= mhclo.mh2gd_index.size():
```

Comment:
This was the second crash fix, in some cases the vertex index was completely missing thus it would assert it could not find the index. Adding a check to ensure it doesn't try is a good idea.

The underlying cause was unclear.